src/App.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import './App.css';
+import Modal from './Components/Modal';
+
+function App() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const appTitle = `Componente modal con React`
+  const buttonTxt = `Abrir`;
+  return (
+    <div className="App">
+      <h1 className='appHeader'>{appTitle}</h1>
+      <button className="openModal" onClick={() => {setIsModalOpen(true);}}>{buttonTxt}</button>
+      {isModalOpen && <Modal setModalOpen={setIsModalOpen}/>}
```

Comment:
Why this instead of: 

```suggestion
  Modal visible={isModalOpen} setModalOpen={setIsModalOpen}/>
```

Do you know the difference between using one or the other?

Reply:
I don't know is there is a more technical answer, but I think using the code that you suggest, you avoid to do the logic to close and open the Modal component in a external component, so I think that it is better becouse you only have to care about send the correct properties to de Modal component and it will do the internal logic.
I use it in my last commit.